.github/workflows/welcome-issue-bot.yml

```
@@ -16,118 +14,41 @@ concurrency:
-            // Prevent duplicate welcome comments
-            const comments =
-              await github.paginate(
-                github.rest.issues.listComments,
-                {
-                  owner,
-                  repo,
-                  issue_number,
-                  per_page: 100
-                }
-              );
+Thanks for opening an issue! We’ll run quick quality + duplicate checks and keep you updated.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)